bin/heudiconv

```diff
@@ -657,26 +668,36 @@ def convert_dicoms(subjs, dicom_dir_template, outdir, heuristic_file, converter,
                 "atm we support processing only 1 subject/session at a time"
             sess, sid = session_subject_ids.pop()
 
+
+        if is_bids and not sid.isalnum(): # alphanumeric only
+            old_sid = sid
```

Comment:
I'll reference the phantom conversion - the subject ID is `PHANTOM1_3`, but having `sub-PHANTOM1_3_func...` would mess up BIDS's separation by underscore. Rather than throwing an error and forcing you to rename the directory where the dicoms are, we can just drop the underscore and throw a warning that it was renamed. Really, this just makes it easier to do the demonstration next week